events/elections/2021/election.yaml

```diff
@@ -1,7 +1,7 @@
 name: 2021 Steering Committee Election
 organization: Kubernetes
-start_datetime: 2021-10-07 00:00:01
-end_datetime: 2021-10-28 23:59:00
+start_datetime: 2021-10-14 00:00:01
+end_datetime: 2021-110-04 23:59:00
```

Comment:
```suggestion
end_datetime: 2021-11-04 23:59:00
```

Reply:
wait, I just pushed a fix for that?  WTF?

Reply:
@jberkus Made the comment before you pushed the fix. :-)